### ISSUE TITLE
fmt-0.6.1.2 test suite fails with newline-related errors

### ISSUE THREAD

**Issue by DanBurton** [cdornan/fmt#30]
It's a bunch of failures like this:

```
      tests/Main.hs:681:7:
      21) formatters.'genericF', custom types, records
           expected: "Qux:\n  q1: 1\n  q2: True\n  q3: hi"
            but got: "Qux:\n  q1: 1\n  q2: True\n  q3: hi\n"
```

Where the only thing wrong is a trailing newline.

**Comment by DanBurton** [comment]
I was able to reproduce this locally like so:

```bash
stack unpack fmt-0.6.1.2 && cd fmt-0.6.1.2
echo 'resolver: nightly-2020-07-04' > stack.yaml
stack build --test --bench --no-run-benchmarks --fast
```

**Comment by DanBurton** [comment]
I believe this is due to changes in `neat-interpolation`. see: https://hackage.haskell.org/package/neat-interpolation-0.5.1/changelog

**Comment by cdornan** [comment]
@DanBurton agree with all of the above &mdash; have a fix and will relocate fmt in ghc-9 test failures section (for the doctests).

**Comment by cdornan** [comment]
fixed by e0e1e98dfe35486d6f1ea1b2389bd4af4d93914c
